src/analysis/classic/warlock/demonology/modules/spells/Doomguard.tsx

```diff
@@ -0,0 +1,394 @@
+            this.snapshotQualityEntries.push({
+              item: equippedItem,
+              snapshotQuality: QualitativePerformance.Fail,
+              relatedBuffs: [engineering.SYNAPSE_SPRINGS_INTEL_BUFF],
+              issueWithItemOrEnchant: !synapseSpringsAvailable,
+              snapshotSummary: synapseSpringsAvailable ? (
+                <></>
+              ) : (
+                <li>
+                  <ItemLink id={equippedItem.id} details={equippedItem} icon={false}>
+                    <Icon icon={equippedItem.icon} />
+                  </ItemLink>{' '}
+                  is not enchanted with <SpellLink spell={engineering.SYNAPSE_SPRINGS} />. If you
+                  are not engineer, consider to change profession. This is by far the best one,
+                  especially as a warlock due to snapshot mechanics.
+                </li>
+              ),
+            });
+          }
+          break;
+        case GEAR_SLOTS.BACK:
+          {
+            const lighweaveAvailable =
+              equippedItem.permanentEnchant === tailoring.LIGHTWEAVE_BUFF_RANK_2.enchantId;
+            this.snapshotQualityEntries.push({
+              item: equippedItem,
+              snapshotQuality: QualitativePerformance.Fail,
+              relatedBuffs: [tailoring.LIGHTWEAVE_BUFF_RANK_2],
+              issueWithItemOrEnchant: !lighweaveAvailable,
+              snapshotSummary: lighweaveAvailable ? (
+                <></>
+              ) : (
+                <li>
+                  <ItemLink id={equippedItem.id} details={equippedItem} icon={false}>
+                    <Icon icon={equippedItem.icon} />
+                  </ItemLink>{' '}
+                  is not enchanted with <SpellLink spell={tailoring.LIGHTWEAVE_BUFF_RANK_2} />. If
+                  you are not tailor, consider to change profession. This is the 2nd best
+                  profession, especially as a warlock due to snapshot mechanics.
+                </li>
+              ),
+            });
+          }
+          break;
+        case GEAR_SLOTS.MAINHAND:
+          {
+            const powerTorrentAvailable =
+              equippedItem.permanentEnchant === enchanting.POWER_TORRENT_BUFF.effectId;
+            this.snapshotQualityEntries.push({
+              item: equippedItem,
+              snapshotQuality: QualitativePerformance.Fail,
+              relatedBuffs: [enchanting.POWER_TORRENT_BUFF],
+              issueWithItemOrEnchant: !powerTorrentAvailable,
+              snapshotSummary: powerTorrentAvailable ? (
+                <></>
+              ) : (
+                <li>
+                  <ItemLink id={equippedItem.id} details={equippedItem} icon={false}>
+                    <Icon icon={equippedItem.icon} />
+                  </ItemLink>{' '}
+                  is not enchanted with <SpellLink spell={enchanting.POWER_TORRENT_BUFF} />. This is
+                  by far the best weapon enchant, especially as a warlock due to snapshot mechanics.
```

Comment:
probably can skip explaining that the only relevant caster enchant is the best one.

```suggestion
                  is not enchanted with <SpellLink spell={enchanting.POWER_TORRENT_BUFF} />.
```